tests/integration/python/test_python.py

```diff
@@ -20,16 +20,27 @@ def models(self):
     def project_config(self):
         return {
             "config-version": 2,
-            "vars": {"http_path": os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH")},
+            "vars": {
+                "http_path": os.getenv("DBT_DATABRICKS_CLUSTER_HTTP_PATH"),
+                "location_root": "",
+            },
         }
 
     def python_exc(self):
-        self.run_dbt(["run"])
+        self.run_dbt(["run", "-s", "basic"])
 
     @use_profile("databricks_sql_endpoint")
     def test_python_databricks_sql_endpoint(self):
         self.python_exc()
 
     @use_profile("databricks_uc_sql_endpoint")
     def test_python_databricks_uc_sql_endpoint(self):
+        self.use_default_project(
+            {
+                "vars": {
+                    "http_path": os.getenv("DBT_DATABRICKS_UC_CLUSTER_HTTP_PATH"),
+                    "location_root": "",
```

Comment:
So, this was a change I needed to make in order to get the two tests to pass when I had a UC cluster and a non-UC cluster.  My understanding is that they were passing before because the cluster that was pointed at by DBT_DATABRICKS_CLUSTER_HTTP_PATH in the integration environment (and you and Andre's local test env) was UC enabled.